README.md

```diff
@@ -268,5 +268,5 @@ license.
 
 See the LICENSE file for more details.
 
-The remaining code is Copyright 2012-2020 the GoSNMP Authors - see
+The remaining code is Copyright 2012-2021 the GoSNMP Authors - see
```

Comment:
We can remove the ending date for this entry.

```suggestion
The remaining code is Copyright 2012-2021 the GoSNMP Authors - see
```
```suggestion
The remaining code is Copyright 2012 the GoSNMP Authors - see
```

Reply:
Done